package.json

```diff
@@ -62,6 +62,11 @@
           "type": "boolean",
           "description": "Synchronize dependency viewer selection with folder explorer",
           "default": true
+        },
+        "java.dependency.isHierarchicalView": {
```

Comment:
Should avoid use this value.  It is not clear to the end user what the opposite is? 

Should be enum values: "flat", "hierarchy"

Reply:
Resolved